docs/src/assets/scss/components/copy-button.scss

```diff
@@ -0,0 +1,48 @@
+[class*="language-"] > .c-btn--copy {
+    direction: ltr;
+    position: absolute;
+    top: 12px;
+    right: 12px;
+    z-index: 3;
+    border-radius: 4px;
+    width: 40px;
+    height: 40px;
+    background-color: var(--copy-button-background-color);
+    cursor: pointer;
+    background-image: url("data:image/svg+xml,%3Csvg width='20' height='20' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M4.167 12.5h-.834a1.667 1.667 0 0 1-1.666-1.667v-7.5a1.667 1.667 0 0 1 1.666-1.666h7.5A1.666 1.666 0 0 1 12.5 3.333v.834M9.167 7.5h7.5c.92 0 1.666.746 1.666 1.667v7.5c0 .92-.746 1.666-1.666 1.666h-7.5c-.92 0-1.667-.746-1.667-1.666v-7.5c0-.92.746-1.667 1.667-1.667Z' stroke='#667085' stroke-width='1.667' stroke-linecap='round' stroke-linejoin='round'/%3E%3C/svg%3E");
+    background-position: 50%;
+    background-size: 20px;
+    background-repeat: no-repeat;
+    transition:
+        border-color 0.25s,
+        background-color 0.25s,
+        opacity 0.25s;
+}
+
+[class*="language-"] > .c-btn--copy:hover,
+[class*="language-"] > .c-btn--copy.copied {
+    opacity: 0.8;
```

Comment:
The semi-transparency is a problem when the button overlaps text, making it harder to see the button on hover.

What do you think about doing a box-shadow on hover instead?